comp/otelcol/collector-contrib/impl/go.mod

```diff
@@ -7,12 +7,14 @@ replace github.com/DataDog/datadog-agent/comp/otelcol/collector-contrib/def => .
 require (
 	github.com/DataDog/datadog-agent/comp/otelcol/collector-contrib/def v0.54.0-rc.2
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/countconnector v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector v0.98.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/exceptionsconnector v0.98.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/grafanacloudconnector v0.98.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/routingconnector v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/connector/servicegraphconnector v0.100.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector v0.98.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.98.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.100.0
```

Comment:
should we update all of them to v0.100.0 ?

Reply:
Perhaps as a later change, right now I'm just trying to get the disabled components to be reenabled. Updating version numbers for everything requires an unpredictable amount of work, due to the need to get dependencies to work together, and licenses and copyrights to be correct, such that the linters are happy.